README.md

```diff
@@ -26,17 +26,18 @@ When **creating a new major branch for a pre-release major version**
 
 - Make sure you've added a new major branch to both `silverstripe/developer-docs` and `silverstripe/silverstripe-userhelp-content`
 - Add the new major to `sources-docs.js` and `sources-user.js`
-- Add new major branches for various modules as defined in `sources-docs.js` and `sources-user.js` as well
+- Add the new major branches for various modules as defined in `sources-docs.js` and `sources-user.js` as well
 - Add the new major to the version select in `src/components/Header.tsx`
-- Add the major to the `PRE_RELEASE` array in the `getVersionMessage` function in `src/utils/nodes.ts`
+- Add the new major to the `PRE_RELEASE` array in the `getVersionMessage` function in `src/utils/nodes.ts`
 - Add the new major version to the [algolia crawler script](https://crawler.algolia.com/admin/crawlers/3d14ccdd-f9ae-4957-bc0a-5b21b4c29af3/configuration/edit)
 
 For **new stable releases**, you will need to do the following
 
 - Remove the major from the `PRE_RELEASE` array in the `getVersionMessage` function in `src/utils/nodes.ts`
 - Update the `getDefaultVersion` function's return value to the new stable major in `src/utils/nodes.ts`
 - Update redirects in `netlify.toml`
-- Update `index.tsx` to navigate to the new stable major
+- Update `src/pages/index.tsx` to navigate to the new stable major
+- Add the new major to `redirects` in `gatsby-node.js`
```

Comment:
This step was missing and is necessary for e.g. `https://docs.silverstripe.org/en/getting_started/environment_management/` to get redirected to `https://docs.silverstripe.org/en/6/getting_started/environment_management/`